backend/src/email/routes/email-campaign.routes.ts

```diff
@@ -185,6 +185,12 @@ export const InitEmailCampaignRoute = (
     emailMiddleware.duplicateCampaign
   )
 
+  router.post(
+    '/select-phonebook-list',
+    celebrate(selectListValidator),
+    emailTemplateMiddleware.selectPhonebookListHandler
```

Comment:
nit: this api path  sounds more user action than an api path

Reply:
sorry i don't quite understand, it IS a user action, to select a phonebook list. 
This is to keep things consistent with the older `managed-list` functionality, whose API was named as `select-list`

Reply:
User action - Get lists -->  translated to Get /lists
User action - create list --> translated to POST /list

>This is to keep things consistent with the older managed-list functionality, whose API was named as select-list